### PR TITLE
Add Support for Hindi

### PR DESCRIPTION
**Pull request by aadityataparia**
Added hindi transliterator and spec.

Basis for these characters - http://www.utf8-chartable.de/unicode-utf8-table.pl?start=2304&number=128 This is list for all devanagri (writing script for hindi) characters present in UTF-8

Hindi used in spec -
आदित्य तापड़िया (Aaditya Taparia)
सबरीमाला करवाना पायसम (Random hindi string)
सक्रांति आँख (Sun's revolution eye)

My transliteration is based on how normally people transliterate hindi to english while chatting/writing online based on pronunciation in India.

### CODE REVIEW

File: spec/spec_helper.rb [hunk 1]
```diff
@@ -33,4 +33,13 @@
     string = "Γερμανία"
     expect(t.transliterate(string)).to match(/[\x0-\x7f]/)
   end
+end
+
+shared_examples_for "a hindi transliterator" do
+  let(:t) { described_class.instance }
+
+  it "should transliterate greek characters" do
```
Comment (davidstosik) [inline spec/spec_helper.rb]: `s/greek/hindi/`

File: lib/babosa/transliterator/base.rb [hunk 2]
```diff
@@ -21,6 +21,7 @@ module Transliterator
     autoload :Ukrainian,  "babosa/transliterator/ukrainian"
     autoload :Greek,      "babosa/transliterator/greek"
     autoload :Vietnamese, "babosa/transliterator/vietnamese"
+    autoload :Hindi,      "babosa/transliterator/hindi"
```
Comment (davidstosik) [inline lib/babosa/transliterator/base.rb]: What do you think of moving this to line 13, in order to keep alphabetic order (except for Greek, which I'd suggest to fix in another PR)?

**Comment by aadityataparia** [comment]
@norman

File: spec/spec_helper.rb (hunk 1 above)
Comment (aadityataparia) [reply spec/spec_helper.rb]: Done

File: lib/babosa/transliterator/base.rb (hunk 2 above)
Comment (aadityataparia) [reply lib/babosa/transliterator/base.rb]: Done

**Comment by aadityataparia** [comment]
@parndt What do you think about forking the repo and make it active again?

**Comment by aadityataparia** [comment]
👍  :D

**Pull request merged**